### PR TITLE
set horizontalPadding to HelpDialog Items

### LINKED ISSUE #14042: [BUG]: Add horizontalPadding to HelpDialog items
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

1. Go to NavigationDrawer
2. Click on Help

### Expected behaviour

To look better, there must be some horizontalPadding to items

### Actual behaviour

No horizontalPadding to HelpDialog items

WhatsApp Image 2023-06-28 at 12 05 27 AM

### Debug info

```text
AnkiDroid Version = 2.16.0beta4-debug

Android Version = 13

ProductFlavor = amazon

Manufacturer = realme

Model = RMX3521

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 13; RMX3521 Build/RKQ1.211119.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/114.0.5735.130 Mobile Safari/537.36

ACRA UUID = 20b2dc80-f22d-4759-9817-396d281f95b3

New schema = false

Scheduler = std2

Crash Reports Enabled = false

DatabaseV2 Enabled = true
```

### (Optional) Anything else you want to share?

_No response_

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by SanjaySargam**
## Purpose / Description
Add horizontalPadding to HelpDialog items

## Fixes
Fixes #14042 

## Approach
set paddingLeft to 32 pixels

## How Has This Been Tested?

Tested on Physical Device (Realme 9
WhatsApp Image 2023-07-02 at 3 10 52 PM

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/RecursivePictureMenu.kt [hunk 1]
```diff
@@ -45,6 +45,7 @@ class RecursivePictureMenu : DialogFragment() {
         val adapter: RecyclerView.Adapter<*> = object : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
             override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
                 val root = layoutInflater.inflate(R.layout.material_dialog_list_item, parent, false)
+                root.setPadding(32.dpToPx(), 0, 0, 0)
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/dialogs/RecursivePictureMenu.kt:48]: There's no need to set the padding manually as the layout file used above has padding set on it(of just 10dp) and it also sets is symmetrical.

So change the padding in the layout file to `32dp`.

**Review by lukstbit — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/RecursivePictureMenu.kt (hunk 1 above)
Comment (SanjaySargam) [reply AnkiDroid/src/main/java/com/ichi2/anki/dialogs/RecursivePictureMenu.kt:48]: Will I change all paddings to 32dp or only paddingLeft?

Comment (lukstbit) [reply AnkiDroid/src/main/java/com/ichi2/anki/dialogs/RecursivePictureMenu.kt:48]: The horizontal padding on both sides, so right and left(I think the IDE will also complain if you don't do it).

Comment (lukstbit) [reply AnkiDroid/src/main/java/com/ichi2/anki/dialogs/RecursivePictureMenu.kt]: @SanjaySargam Also change the paddingStart and paddingLeft to 32dp.

**Review by lukstbit — APPROVED** [review]

**Review by abhineshchandra1234 — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
Nice! Thank you

**Pull request merged**